drone/client.go

```diff
@@ -266,7 +266,11 @@ func (c *client) BuildLast(owner, name, branch string) (*Build, error) {
 	out := new(Build)
 	uri := fmt.Sprintf(pathBuild, c.addr, owner, name, "latest")
 	if branch != "" {
-		uri += "?branch=" + branch
+		if strings.HasPrefix(branch, "refs/") {
+			uri += "?ref=" + branch
```

Comment:
I think it would be cleaner to create a new function BuildLastByRef and call the parameter ref. This change still uses "branch" which isn't as clear to read.

Reply:
Also with this change the comment is no longer correct.